e2e/compilation-cases/package.json

```diff
@@ -11,6 +11,18 @@
     "@types/jquery": "3.5.29",
     "devextreme": "workspace:*",
     "jquery": "3.7.1",
-    "typescript": "4.9.5"
+    "typescript": "4.9.5",
+    "eslint": "catalog:",
+    "@eslint-stylistic/metadata": "catalog:",
```

Comment:
Do we really need this package here? I haven't seen us use it.

Reply:
deleted it